kornia/utils/image.py

```diff
@@ -69,10 +69,12 @@ def _to_bchw(tensor: torch.Tensor, color_channel_num: Optional[int] = None) -> t
     if len(tensor.shape) == 3:
         tensor = tensor.unsqueeze(0)
 
-    if color_channel_num is not None and color_channel_num != 1:
-        channel_list = [0, 1, 2, 3]
-        channel_list.insert(1, channel_list.pop(color_channel_num))
-        tensor = tensor.permute(*channel_list)
+    # TODO(jian): this function is never used. Besides is not feasible for torchscript.
+    # In addition, the docs must be updated. I don't understand what is doing.
+    # if color_channel_num is not None and color_channel_num != 1:
+    #    channel_list = [0, 1, 2, 3]
+    #    channel_list.insert(1, channel_list.pop(color_channel_num))
+    #    tensor = tensor.permute(*channel_list)
```

Comment:
@shijianjian could you verify this ?

Reply:
This is a more general function that in case the input is (B, H, W, C) or other weird formats.

So, if (B, H, W, C), we will need to have "color_channel_num == 3" to convert it (B, C, H, W).

Reply:
Is it really used ? And really needed ? Couldn't raise any breakpoint in tests to verif.  Besides, I think the documentation between 2d and 3d have typo.